src/main/java/org/sagebionetworks/web/client/widget/entity/editor/TableQueryResultWikiViewImpl.java

```diff
@@ -23,10 +23,9 @@ public interface TableQueryResultViewUiBinder extends UiBinder<Widget, TableQuer
 	CheckBox isShowTableOnly;
 
 	@Inject
-	public TableQueryResultWikiViewImpl(TableQueryResultViewUiBinder binder, CookieProvider cookies) {
+	public TableQueryResultWikiViewImpl(TableQueryResultViewUiBinder binder) {
 		widget = binder.createAndBindUi(this);
-		boolean isInTestWebsiteMode = DisplayUtils.isInTestWebsite(cookies);
-		isShowTableOnly.setVisible(isInTestWebsiteMode);
+		isShowTableOnly.setVisible(true);
```

Comment:
This works - alternatively it could be set to visible in the ui.xml instead (and this line removed).